src/ServiceProvider.php

```diff
@@ -5,6 +5,7 @@
 namespace Kreait\Laravel\Firebase;
 
 use Illuminate\Contracts\Container\Container;
+use Illuminate\Contracts\Foundation\Application;
```

Comment:
Same as in `FirebaseProjectManager` - I used `Container` in other parts of the app because of Lumen and if it's possible, we should stick with that